spec/vehicle_train_cars.schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "primaryKey": [
+    "vehicle_id",
+    "train_car_id"
+  ],
+  "missingValues": [
+    "NA",
+    "NaN",
+    ""
+  ],
+  "fields": [
+    {
+      "name": "vehicle_id",
+      "type": "string",
+      "title": "ID referencing vehicles.vehicle_id",
+      "description": "Identifies a vehicle, such as a train consist. If possible, this should match other internal agency vehicle IDs.",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "train_car_id",
+      "type": "string",
+      "title": "ID referencing train_car.train_car_id",
+      "description": "Identifies a train car or an asset that is a component of the vehicle.",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "order",
+      "type": "integer",
+      "description": "The assigned order of the train car or asset within the vehicle.",
+      "constraints": {
+        "minimum": 0
+      }
+    },
+    {
+      "name": "operator_id",
+      "type": "string",
+      "description": "Identifies an operator. If possible, this should match other internal agency operator IDs.",
```

Comment:
Is this a person or an agency? Seems like we wouldn't want to require ids for people just to know the composition of a train.

Reply:
Per the G-18 pre-publication final report, it is a person.
Agree that it shouldn't be required.